doc/languages-frameworks/python.section.md

```diff
@@ -2021,9 +2021,22 @@ The following rules are desired to be respected:
   and using a `-` as delimiter.
 * Attribute names in `python-packages.nix` should be sorted alphanumerically to
   avoid merge conflicts and ease locating attributes.
+* The list of [dependencies to avoid or replace](#python-dependencies-avoid-replace) is respected.
 
 This list is useful for reviewers as well as for self-checking when submitting packages.
 
+## Dependencies to avoid or replace {#python-dependencies-avoid-replace}
+
+Some dependencies do not make sense inside a nix build and should be completely avoided or replaced by a stub.
+Below you find a listing of the most common ones.
+
+| Package | Replacement | Reason |
+|---------|-------------|--------|
+| cmake | cmake (stub) | The package normally downloads cmake from the internet. The stub provides our cmake with the usual setup hook. |
+| ninja | ninja (stub) | The package normally downloads ninja from the internet. The stub provides our cmake with the usual setup hook. |
```

Comment:
This seems more like an implementation detail than actionable advice for package authors, right?

Reply:
Well, jain. It is probably a good hint to kinda know the difference between pkgs.ninja and pkgs.python3Packages.ninja and what to do when you encounter it.

Reply:
For example with this information people know that they don't need to remove the dependency which would otherwise be necessary because downloading things in the sandbox is well known to not work.